wordpress-mobile/WordPress-iOS
# Add feature flag for new login project

New login work can go behind this feature flag until ready for release. Adds a new `Login.storyboard` to hold the new UI. Currently the only difference is use of the initial view controller setting, and a "New Login" label:

![login-feature-flag-ss](https://cloud.githubusercontent.com/assets/517257/24975812/3b4be14c-1f84-11e7-88d9-1cacee8d286c.jpg)

To test:
1. Launch app after a fresh install (no existing accounts in app)
2. Ensure the first screen has the "New Login" label at top

Needs review: @SergioEstevao
